ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSemanticAnalyzer.java

```diff
@@ -372,16 +382,22 @@ private Path getCurrentLoadPath() throws IOException, SemanticException {
           }
         }
         Path hiveDumpPath = new Path(latestUpdatedStatus.getPath(), ReplUtils.REPL_HIVE_BASE_DIR);
-        if (loadPathBase.getFileSystem(conf).exists(new Path(hiveDumpPath,
```

Comment:
why is this check removed?

Reply:
Have moved the check to shouldLoadProceed method